packages/react-internal/src/components/SearchBox/SearchBox.base.tsx

```diff
@@ -79,97 +82,96 @@ export const SearchBoxBase: React.FunctionComponent<ISearchBoxProps> = React.for
     'role',
   ]);
 
-  const onClear = React.useCallback(
+  const _onClear = React.useCallback(
     (ev: React.MouseEvent<HTMLAnchorElement | HTMLButtonElement> | React.KeyboardEvent<HTMLElement>) => {
-      customOnClear?.(ev);
+      onClear?.(ev);
```

Comment:
I'm open to feedback about whether we prefer using a leading underscore (e.g. `_onClear`) for local callbacks, or sticking with renaming the props versions (e.g. `customOnClear`). The underscore seemed to be cleaner and less prone to mistakes.

Reply:
I think in other components we're more commonly using `custom_____` so I'd probably prefer to stick with that convention for now.

Reply:
Though I see your point about how using the underscore would more directly mirror the old convention for private methods...